### PR TITLE
mac os ffmpeg d/l link + instructions

### PR DESCRIPTION
**Pull request by CouldBeThis**
* the download link was broken so I found a working link from https://ffmpeg.org/
* added (hopefully) slightly clearer homebrew instructions 
* added link to source while I was linking things in case for some reason a person who knows what to do with that information wasn't able to find it for themselves

A sentence of explanation as to what ffmpeg does in the application (or what kinds of errors would be expected without it) might be helpful here but I don't know so can't write it.

(I am not very confident about this one; I just intended to fix the link and got carried away)

### CODE REVIEW

File: general/administration/installing.md [hunk 1]
```diff
@@ -428,9 +428,14 @@ Closing the terminal window will end Jellyfin. Running Jellyfin in screen or tmu
...
 **Using FFmpeg with the Portable Version**
 
-The portable version doesn't come with FFmpeg by default. To install FFmpeg you can use homebrew or download the build from [Zeranoe](https://ffmpeg.zeranoe.com/builds/macos64/static/).
+The portable version doesn't come with FFmpeg by default. To install FFmpeg you have 3 options:
+* use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
+* download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
+* compile from source available from the it's [website](https://ffmpeg.org/download.html)
 
-If using Zeranoe's build, extract it to the `/Applications/` folder.
+More detailed download options, documentation and signatures can be found .
```
Comment (dkanada) [inline general/administration/installing.md:436]:
```suggestion
More detailed download options, documentation and signatures can be found.
```

Comment (dkanada) [inline general/administration/installing.md:436]:
```suggestion
More detailed download options, documentation, and signatures can be found.
```

File: general/administration/installing.md [hunk 2]
```diff
@@ -428,9 +428,14 @@ Closing the terminal window will end Jellyfin. Running Jellyfin in screen or tmu
 
 **Using FFmpeg with the Portable Version**
 
-The portable version doesn't come with FFmpeg by default. To install FFmpeg you can use homebrew or download the build from [Zeranoe](https://ffmpeg.zeranoe.com/builds/macos64/static/).
+The portable version doesn't come with FFmpeg by default. To install FFmpeg you have 3 options:
```
Comment (dkanada) [inline general/administration/installing.md:431]:
```suggestion
The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have three options.
```

File: general/administration/installing.md [hunk 3]
```diff
@@ -428,9 +428,14 @@ Closing the terminal window will end Jellyfin. Running Jellyfin in screen or tmu
...
+* use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
+* download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
+* compile from source available from the it's [website](https://ffmpeg.org/download.html)
```
Comment (dkanada) [inline general/administration/installing.md:434]:
```suggestion
* compile from source available from the official [website](https://ffmpeg.org/download.html)
```

**Comment by dkanada** [comment]
The content looks good to me, but our linting reported these errors.

```
general/administration/installing.md:432:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
general/administration/installing.md:432 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "* use the package manager home..."]
general/administration/installing.md:433:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
general/administration/installing.md:434:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
```

**Comment by CouldBeThis** [comment]
thank you @dkanada for the editing. so embarrassed I used the wrong "its"! 

I have never used this system before, I think I did the things it was asking for but if anything missed let me know.

**Comment by nyanmisaka** [comment]
There are some lint errors:
```
general/administration/installing.md:432:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
general/administration/installing.md:432 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "* use the package manager home..."]
general/administration/installing.md:433:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
general/administration/installing.md:434:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]

```

File: general/administration/installing.md [hunk 4]
```diff
@@ -428,9 +428,14 @@ Closing the terminal window will end Jellyfin. Running Jellyfin in screen or tmu
 
 **Using FFmpeg with the Portable Version**
 
-The portable version doesn't come with FFmpeg by default. To install FFmpeg you can use homebrew or download the build from [Zeranoe](https://ffmpeg.zeranoe.com/builds/macos64/static/).
+The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have three options.
+* use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
+* download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
+* compile from source available from the official [website](https://ffmpeg.org/download.html)
```
Comment (dkanada) [inline general/administration/installing.md:434]:
```suggestion
- use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
- download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
- compile from source available from the official [website](https://ffmpeg.org/download.html)
```

File: general/administration/installing.md [hunk 5]
```diff
@@ -428,9 +428,14 @@ Closing the terminal window will end Jellyfin. Running Jellyfin in screen or tmu
 
 **Using FFmpeg with the Portable Version**
 
-The portable version doesn't come with FFmpeg by default. To install FFmpeg you can use homebrew or download the build from [Zeranoe](https://ffmpeg.zeranoe.com/builds/macos64/static/).
+The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have three options.
+- use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
```
Comment (dkanada) [inline general/administration/installing.md:432]:
```suggestion

- use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
```

**Review by dkanada — APPROVED** [review]

**Pull request merged**